src/styles/scss/style.scss

```diff
@@ -144,41 +168,48 @@ body {
 	font-size: 25px;
 	background: #fff;
 	cursor: pointer;
+
 	#sidebar-close {
 		cursor: pointer;
 		color: $color_2;
 	}
 }
+
 .main {
 	left: 260px;
+	top: 55.98px;
 	width: calc(100% - 260px);
 	transition: all 0.5s ease;
 	z-index: 1000;
 	position: relative;
 	display: flex;
 	align-items: center;
 	justify-content: center;
-	height: 100vh;
```

Comment:
Commented this part to allow scrollable content that is not limited to the screen height
Kindly check if it will affect negatively to other pages Thank you